---
title: "Handling biomedical images and sharing reproducible workflows"
subtitle: "A primer for researchers"
author: 
  - name: FRDR curation team
    email: curators@frdr-dfdr.ca
    affiliations:
      - name: Digital Research Alliance of Canada
format: 
 
  html:
    embed-resources: true
    mermaid:
      theme: forest
  
  revealjs:
    footer: "Sharing images and reproducible workflows - FRDR curation team"
    logo: "images/alliance_logo.png"
    theme: white
    transition: fade
    slide-number: true
    progress: true
    show-slide-number: all
    preview-links: auto
    resources:
      - images/Presentation_QR-code.png
    mermaid:
      theme: forest
      mermaid-format: png
    revealjs-plugins:
      - kapsner/audio-slideshow
    revealjs-args:
      - audio: true
      - audioAutoplay: false
      - audioControls: true
    
filters:
  - reveal-auto-agenda
auto-agenda:
  bullets: numbered
  heading: Agenda

css: styles.css
bibliography: references.bib 
editor: source
---

# Understanding BioImaging

## 

::: {style="text-align: center;font-size: 60%"}
![](images/ImagingSamples.jpg){fig-alt="A diagram illustrating different types of biological imaging techniques and their applications. The top section, labeled 'Sample Imaging,' includes icons representing various research fields such as molecular biology, genetics, microbiology, and anatomy. Below, three imaging modalities are depicted: 'Electron microscopy' (in light blue), 'Light microscopy' (in dark blue), and 'Human bioimaging' (in dark red), indicating their areas of application. At the bottom, 'Image data analysis' is shown, emphasizing the computational aspect of image processing. The diagram highlights the role of imaging across multiple research disciplines." fig-align="center" width="100%"}
:::

## Images contain countless data{.center}

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 80%"}
![Henning Falk (2022) - NumFOCUS](images/DataTheraphy.png){fig-alt="A black-and-white cartoon illustration depicting a therapy session. A patient is lying on a couch, expressing their concerns to a therapist who is taking notes. The patient has a thought bubble showing a massive wave labeled 'DATA' about to crash over a small, helpless figure. The cartoon humorously represents the overwhelming feeling of dealing with large amounts of data, a common challenge in research and data science. The drawing is signed by Henning Falk." fig-align="center" width="65%"}
:::
::::

:::: {#second-column}
::: {style="text-align: left;font-size: 80%"}
Bioimaging has entered the realm of big data comprising [increasingly complex datasets]{style="color:green;"}. We face numerous challenges, specifically, proper [data handling]{style="text-decoration: underline;"} and [management]{style="text-decoration: underline;"} and the creation and sharing of [reproducible image analysis]{style="text-decoration: underline;"} workflows.
:::
::::
:::::

## BioImage Life cicle

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 80%"}
![BioImage life cycle from [Schmidt, et al. 2024](https://doi.org/10.1111/jmi.13317)](images/ImageLifeCycle.jpg){fig-alt="A circular diagram illustrating the bioimaging research data management (RDM) lifecycle, titled 'Core Facility: Infrastructure and Bioimaging RDM Support.' At the center, a researcher icon represents the user interacting with different stages of the workflow. The lifecycle is divided into segments: 'Image Acquisition' (purple), 'Image File Storage & Access' (yellow), 'Processing & Bioimage Analysis' (green), '(Data-) Publication' (blue), 'Archiving, Long-Term Storage' (light gray), and 'Planning, Data Search & Re-use' (gray). Around the outer circle, key supporting infrastructure elements are represented, including cloud storage, file system storage, efficient data transfer, formats, standards, tools, permission systems, data security, web access, connectivity, and data integration. The diagram highlights the structured approach to managing and supporting bioimaging research data." fig-align="center" width="80%"}
:::
::::

:::: {#second-column}
::: {style="text-align: left;font-size: 90%"}
BioImages have the potential for scientific discovery beyond its original acquisition purpose when handled according to [FAIR principles]{style="color:green;"} ([Schmidt, et al. 2024](https://onlinelibrary.wiley.com/doi/10.1111/jmi.13317); see [Wilkinson, et al. 2016](https://pmc.ncbi.nlm.nih.gov/articles/PMC4792175/)).
:::
::::
:::::

# Planning and adquiring images

## Overview {.center}

::: {style="text-align: left;font-size: 80%"}
Conventional file-system-based storage is quickly reaching its [limits]{style="color:red;"}. Before the data are generated, researchers must consider how it will be [stored]{style="text-decoration: underline;"}, [moved]{style="text-decoration: underline;"}, [documented]{style="text-decoration: underline;"} and [analyzed]{style="text-decoration: underline;"} during (and after) the projectâ€™s lifetime.
:::

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 80%"}
![](images/Overcapacity.png){fig-alt="A screenshot of the Windows File Explorer 'This PC' section, displaying storage devices and their available disk space. The image highlights two key areas: (1) the 'This PC' navigation option on the left panel, and (2) the 'Devices and drives' section, showing multiple hard drives and partitions, some nearly full with red-colored storage bars. This image illustrates disk space management challenges and overcapacity warnings in Windows." fig-align="center" width="80%"}
:::
::::

:::: {#second-column}
::: {style="text-align: left;font-size: 60%"}

**Current directions:**

- Leverage on research data management plans ([DMP]{style="color:green;"})
- Cloud-based storage and platforms (i.e OMERO) will become a [MUST]{style="color:green;"}.
- Need for standardized formats ([NGFFs]{style="color:green;"}).
:::
::::
:::::

## Planning imaging experiments{.center}  

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
### {{< bi gear-fill >}} Imaging Hadware
::: {style="text-align: left;font-size: 80%"}
- Imaging device (light, confocal, electron). 
- Define imaging parameters - suitable for image analysis.
:::
::::

:::: {#second-column}
### {{< bi database-fill >}} storage of images
::: {style="text-align: left;font-size: 80%"}
-   Image size - Gigas or terabytes.
-   Image complexity - Single or multiple focal planes, Live or time-lapse images.
:::
::::
:::::

:::: {style="text-align: left;font-size: 80%"}
::: callout-warning
## How will you handle your images?
While panning an imaging experiment, consider the most [suitable approach]{style="text-decoration: underline;"} given your capabilities for [data handling and analysis]{style="color:green;"}. 
:::
::::


## Data adquisition{.center}  
::: {style="text-align: left;font-size: 80%"}
- {{< bi gear-fill >}} Desirable instrument [setup and calibration]{style="color:green;"} and [imaging parameters]{style="color:green;"} (size, bit depth, saturation, etc.).
- {{< bi card-image >}} Prioritize [open file formats]{style="color:green;"} (.TIFF) over proprietary (.CZI).
- {{< bi database-fill >}} Recording of [metadata]{style="color:green;"}.
:::

:::: {style="text-align: left;font-size: 80%"}
::: callout-tip
## Adquisition parameters are key
During imaging it is necessary to consider a compromise between the [parameters]{style="color:green;"} needed to answer the research question (magnification, size, bit depth) and the [available processing power]{style="color:green;"} (storage, computing power).
:::
::::

## What parameters should I use to answer my research question?{.center} 
::::: {layout-ncol="2"}
:::: {#first-column}
::: {style="text-align: center;font-size: 60%"}
![8 bit image - Courtesy of Daniel Manrique-Castano](images/Image_8bit.png){fig-alt="A fluorescence microscopy image of a coronal mouse brain section displayed in an image viewer. The image is captured from a `.czi` file and shown as an 8-bit TIFF format with a size of 23MB. The fluorescence signal appears in green, highlighting specific structures within the brain tissue. The metadata in the image viewer indicates dimensions of 9086.62 x 7290.43 microns, corresponding to a resolution of 2752 x 2208 pixels. This type of imaging is commonly used in neuroscience research to visualize protein expression, cellular structures, or anatomical regions." width="100%"}
:::
::::

:::: {#second-column}
::: {style="text-align: center;font-size: 60%"}
![16 bit image - Courtesy of Daniel Manrique-Castano](images/Image_16bit.png){fig-alt="The image shows the same as the left one for a 16-bit depth." width="100%"}
:::
::::
:::::

## Consider that...{.center}

::: {style="text-align: center;font-size: 60%"}
![](images/Microscope.jpeg){fig-alt="A humorous research-themed illustration combining text and a drawing of a microscope. On the left side, bold black text reads 'BRO, DO YOU EVEN SCOPE?' with an illustration of a compound microscope below it. On the right side, within a light blue background, the text emphasizes the importance of training in microscopy: 'A microscope is as good as the person that operates it. Experiment quality can be boosted with training.' The words 'microscope' and 'Experiment quality' are highlighted in blue. The image humorously promotes proper training for obtaining high-quality research results using microscopes." fig-align="center" width="100%"}
:::

## Transforming file formats {.center}

::: {style="text-align: left;font-size: 80%"}
After acquiring original images in [proprietary formats]{style="color:red;"} (i.e .CZI or .LIF), researchers can use different tools to open and transform the images to [open formats]{style="color:green;"} (.TIFF).
::: 

::::: {layout-nrow="2" layout-valign="center"}
:::: {#first-row}
::: {style="text-align: center;font-size: 70%"}
![Bio-Formats from [Open Microscopy Environment](https://www.openmicroscopy.org/bio-formats/downloads/) (compatible with [FIJI](https://imagej.net/formats/bio-formats))](images/Bioformats-Omero.png){fig-alt="A screenshot of the Bio-Formats 8.1.0 download page, a software tool for reading and converting life sciences image file formats. The background contains a fluorescence microscopy image of cells stained with blue, green, and yellow markers. The main heading, 'Bio-Formats 8.1.0 Downloads,' is displayed in large white text. Bio-Formats is an essential tool for researchers working with multidimensional microscopy data." fig-align="center" width="70%"}
:::
::::

:::: {#second-row}
::: {style="text-align: center;font-size: 70%"}
![Bio-Formats for [Python](https://pypi.org/project/python-bioformats/)](images/Bioformats-python.png){fig-alt="A screenshot of the 'python-bioformats 4.1.0' package release page. The page has a blue background and prominently displays the package name and version. Below, a command snippet shows how to install the package using pip: 'pip install python-bioformats,' with a copy button next to it. This package enables Python users to work with Bio-Formats, a library for reading and processing research image data." fig-align="center" width="70%"}
:::
::::
:::::

## The OME.TIFF format{.center}

::: {style="text-align: left;font-size: 80%"}
Is a format developed by the [Open Microscopy Environment (OME)](https://www.openmicroscopy.org/) based on TIFF specification. OME-TIFF incorporates:

- [OME-XML](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2005-6-5-r47) metadata. 
- Support multidimensional, multi image data (see [examples](https://downloads.openmicroscopy.org/images/OME-TIFF/2015-01/bioformats-artificial/)).
- Supports complex multidimensional and high-content screening data using [BigTiff](http://bigtiff.org/), given the limitations of TIFF (up to 4GB).
- Can distribute the data across multiple files (see [example](https://downloads.openmicroscopy.org/images/OME-TIFF/2015-01/tubhiswt-4D/)).
:::

## Next-Generation File Format OME-Zarr{.center}

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 60%"}
![Image Pyramid](images/ImagePyramid.jpeg){fig-alt="A schematic illustration of an image pyramid, a concept used in image processing and bioimaging. The pyramid represents different resolution levels of the same image, with a pink-stained brain tissue sample shown at each level. The base of the pyramid represents 'Full resolution, Full area,' while the middle and upper levels correspond to '1/2 resolution, 1/4 area' and '1/4 resolution, 1/16 area,' respectively. This visualization highlights how images are stored at multiple scales to optimize processing and analysis in whole-slide imaging and multi-resolution microscopy." fig-align="center" width="100%"}
:::
::::

:::: {#second-column}
::: {style="text-align: left;font-size: 70%"}
- Efficient storage and display of the image (Efficient slicing at arbitrary angles).
- Compatible with [Fiji](https://github.com/saalfeldlab/n5-viewer), [Napari](https://github.com/ome/napari-ome-zarr), [ITK/VTK Viewer](https://kitware.github.io/itk-vtk-viewer/docs/index.html), Google's [Neuroglancer](https://github.com/google/neuroglancer?tab=readme-ov-file), [Vizarr](https://github.com/hms-dbmi/vizarr), and Allen Institute's [3D Cell Viewer](https://www.allencell.org/3d-cell-viewer.html).
- For Conda-based batch conversion to OME-Zarr visit [this repository](https://github.com/Euro-BioImaging/BatchConvert).
- Handle [OME-Zarr]((https://link.springer.com/article/10.1007/s00418-023-02209-1)) datasets in Python using [this tool](https://github.com/Euro-BioImaging/ome_zarr_pyramid).
:::
::::
:::::

## Take home message{.center}

::: {style="text-align: left;font-size: 100%"}
Strategic [planing and data management]{style="color:green;"}, cloud-ready [platforms and formats]{style="color:green;"}, and institutional support ensure scalable, accessible, and [reusable]{style="text-decoration: underline;"} imaging data.
:::

# Gathering image metadata

## Overview

[High-quality metadata]{style="color:green;"} is crucial for making imaging data **FAIR**.


::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 80%"}
![Image from [DevianArt](https://www.deviantart.com/bonsaiartistry/art/Fragmented-Whispers-Abstract-Collage-Rhapsody-978347147)](images/Fragmented.png){fig-alt="A vibrant abstract digital artwork featuring a fragmented human face interwoven with geometric shapes, colorful textures, and technological elements. The composition includes a large, expressive eye surrounded by streaks of color, circuits, and intersecting lines. Additional elements, such as a stylized mouth and curved, ribbon-like structures, contribute to the surreal aesthetic." fig-align="center" width="80%"}
:::
::::

:::: {#second-column}
::: {style="text-align: left;font-size: 80%"}
[Current challenges:]{style="color:red;"}

- A fragmented ecosystem [lacking clear metadata standards]{style="color:red;"} that leads to inconsistent documentation.
- Journals and institutions lacking uniform guidelines.
:::
::::
:::::

## Recommended Metadata for Biological Images (REMBI){.center}

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 80%"}
![Image from [Sarkans, et al. 2021](https://doi.org/10.1038/s41592-021-01166-8)](images/Rembi.webp){fig-alt="A Venn diagram illustrating the overlapping metadata needs of different research domains. The three primary categories are 'Research biologistâ€™s needs' (blue), 'Computer vision researcherâ€™s needs' (green), and 'Imaging scientistâ€™s needs' (yellow). Their intersection highlights the 'Recommended metadata,' which serves as a standardized set of information beneficial across these disciplines." fig-align="center" width="70%"}
:::
::::

:::: {#second-column}
::: {style="text-align: left;font-size: 80%"}
[REMBI](https://www.nature.com/articles/s41592-021-01166-8) advice metadata guidelines to address the needs of diverse biomedical imaging communities. 

A summary template is available [here](https://github.com/Alliance-RDM-GDR/RDM_BioimageFAIR/blob/main/resources/REMBI_Overview.xlsx). 

:::
::::
:::::

## 

::: {style="text-align: center;font-size: 80%"}
![[Download](https://github.com/Alliance-RDM-GDR/RDM_BioimageFAIR/blob/main/resources/REMBI_Overview.xlsx) the template](images/Rembi.png){fig-alt="A structured table outlining the different metadata modules required for standardized bioimaging data documentation. The table consists of several sections: 'Study,' 'Study Component,' 'Biosample,' 'Specimen,' 'Image Acquisition,' 'Image Data,' 'Image Correlation,' and 'Analyzed Data.' Each row specifies an attribute (e.g., study type, imaging method, biological entity), a description of its significance, the data entry method (text, ontology, extracted data), and related ontologies (such as EDAM-BIOIMAGING, FBbi, OME, and EFO). This table highlights essential metadata elements needed to ensure reproducibility, interoperability, and structured data sharing in bioimaging research." fig-align="center" width="60%"}
:::

## Use controlled vocabulary{.center}

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
Use [controlled vocabulary]{style="color:green;"} (ontologies) to specify objects, their categories and relationships. 
::::

:::: {#second-column}
::: {style="text-align: left; font-size: 80%"}
| **Name**        | **Ontology**                                       |
|-----------------|----------------------------------------------------|
| Organisms       | [NCBI](Widely supported, non-proprietary)          |
| Genes           | [NCBI](https://www.ncbi.nlm.nih.gov/gene/)         |
| Proteins        | [Uniprot](https://www.uniprot.org/)                |
| Imaging Methods | [FBBI](https://www.ebi.ac.uk/ols4/ontologies/fbbi) | 
| Exp. factors    | [EFO](https://www.ebi.ac.uk/ols4/ontologies/efo)   |
:::
::::
:::::

## Web-based ontology services {.center}

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 80%"}
![[EMBL-EBI Ontology Lookup Service](https://www.ebi.ac.uk/ols4/)](images/OLS.png){fig-alt="Screenshot of the EMBL-EBI Ontology Lookup Service (OLS) homepage. The interface provides a search bar for finding biomedical ontology terms, with options to filter by exact match, include obsolete terms, and include imported terms. Below, there are three sections: 'About OLS,' describing its function as a repository for biomedical ontologies; 'Related Tools,' mentioning additional ontology mapping services like OxO and ZOOMA; and 'Report an Issue,' directing users to a GitHub issue tracker for feedback and ontology requests." fig-align="center" width="100%"}
:::
::::

:::: {#second-column}
::: {style="text-align: center;font-size: 80%"}
![[Biportal](https://bioportal.bioontology.org/)](images/Bioportal.png){fig-alt="Screenshot of Bioportal. This is a comprehensive repository of biomedical ontologies. The interface provides a search bar for finding biomedical ontology terms, with options for advanced search. It shows also a graphs depicting visits and key site statistics." fig-align="center" width="100%"}
:::
::::
:::::

## Record metadata from imaging devices{.center} 
::: {style="text-align: center;font-size: 90%"}
![MicroMeta App](images/MicroMetaApp.png){fig-alt="Screenshot of the Micro Meta App, a microscopy metadata management tool. The top section displays the app's banner with the title 'MICRO META APP' and the tagline 'Microscopy Metadata for the Real World.' Below, two main sections allow users to 'Manage Instrument' and 'Manage Settings,' providing options to document hardware components and acquisition settings. The lower half of the image shows detailed interface views, including labeled microscope components such as objectives, cameras, filter sets, and laser settings, highlighting metadata fields like manufacturer, gain, magnification, and bit depth." fig-align="center" width="70%"}
:::

::: {style="text-align: left;font-size: 80%"}
Visit the [MicroMeta App](https://wu-bimac.github.io/MicroMetaApp.github.io/) and the associated [research article](https://www.nature.com/articles/s41592-021-01315-z)
:::

## Methods J2 PlugIn{.center}

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 80%"}
![Example from MethodsJ2 [repository](https://github.com/ABIF-McGill/MethodsJ2?tab=readme-ov-file)](images/MethodsJ2.png){fig-alt="Screenshot of a text document generated using the MethodsJ2 tool based on user input and a Micro-Meta App hardware file. The document details the imaging setup, including a Zeiss Axiovert 200M inverted microscope configured for Widefield Epifluorescence microscopy, controlled with Zen software. The text specifies the optical components, imaging parameters such as voxel size (0.14 Î¼m), and details of fluorescence excitation using an X-Cite 120 LED light source with various excitation/emission filters and dichroic mirrors. The acknowledgments section credits the Advanced BioImaging Facility (ABIF) at McGill and Joel Ryan for assistance." fig-align="center" width="100%"}
:::
::::

:::: {#second-column}
::: {style="text-align: left;font-size: 80%"}
The MethodsJ2 Fiji plugIn [generates text]({style="color:green;"}) for microscopy [materials and methods]{style="text-decoration: underline;"} by extracting information from metadata (MicroMeta App file). Visit here the [GitHub repository](https://github.com/ABIF-McGill/MethodsJ2?tab=readme-ov-file) or the associated [research article](https://www.nature.com/articles/s41592-021-01290-5). 

:::
::::
:::::


## OMERO: One of the best choices{.center}

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 80%"}
![From [Kunis et al., 2021](https://www.nature.com/articles/s41592-021-01288-z)](images/Omero_Metadata.jpg){fig-alt="Diagram illustrating the workflow of metadata handling in OMERO, from image acquisition on a microscope to metadata import and annotation. The process involves vendor image containers being imported via the OMERO importer, where metadata is extracted using Bio-Formats. Metadata is structured following facility specifications and ontology standards. The OMERO.mde system organizes metadata into structured key-value pairs, enabling visualization, annotation, and export in formats such as .txt, .tsv, and .csv for reuse in IDR format." fig-align="center" width="100%"}
:::
::::


:::: {#second-column}
::: {style="text-align: left;font-size: 80%"}
OMERO incorporates [MDEmic](https://www.nature.com/articles/s41592-021-01288-z) (MetaData Editor for microscopy) an tool that provides an easy way to explore and edit metadata from images.
:::
::::
:::::

## Metadata to consider{.center}

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 60%"}
![From  [Schmied et al., 2023](https://www.nature.com/articles/s41592-023-01987-9)](images/CheckList_ImagesA.png){fig-alt="Checklist for image publishing, covering aspects of image format and image colors/channels. The checklist includes requirements for focusing on relevant content (cropping, rotation, resizing), separating individual images, and providing example images for quantification. It also emphasizes annotation visibility, brightness/contrast adjustments, consistent image comparisons, and color accessibility for color-blind individuals. Recommended steps include adding grayscale versions, intensity scales, and adjustments like gamma correction and pseudocoloring." fig-align="center" width="90%"}
:::
::::

:::: {#Second-column}
::: {style="text-align: center;font-size: 60%"}
![From  [Schmied et al., 2023](https://www.nature.com/articles/s41592-023-01987-9)](images/CheckList_ImagesB.png){fig-alr="Checklist for image annotation and availability. This section highlights the importance of including scale information (such as scale bars and image length in figure legends), explaining all annotations, ensuring legibility of annotations (line width, size, color), and ensuring that annotations do not obscure key data. It also recommends adding relevant imaging details like pixel size, time intervals, and exposure time.The image availability section suggests sharing images with lossless compression, making image files freely downloadable in public databases, and storing them in dedicated image archives." fig-align="center" width="90%"}
:::
::::
:::::

## We need to move toward:{.center}

- Better consolidation of metadata [standards]{style="color:green;"}.
- Development of [tools]{style="color:green;"} for automated metadata collection (includes collaboration with microscope manufacturers).
- Research infrastructures and BioImaging support for proper [metadata recording]{style="color:green;"}.


# Organizing and sharing image data

## Images could be in many places{.center} 

::::: {layout-ncol="3"}
:::: {#first-column}
::: {style="text-align: center;font-size: 60%"}
![Laptop of students and postdocs](images/Laptop.jpg){fig-alt="A laptop with a black screen placed on a wooden table in a cafÃ© setting. A cup of coffee on a saucer is next to it. The background has a wire mesh window with plants visible outside. " fig-align="center" width="100%"}
:::
::::

:::: {#second-column}
::: {style="text-align: center;font-size: 60%"}
![Institute network](images/Network.jpg){fig-alt="A close-up of a network switch with multiple Ethernet cables plugged in. The cables are orange and blue, neatly organized and connected to the switch. A label with the text 'NorthC' is visible on the top right corner of the device." fig-align="center" width="100%"}
:::
::::

:::: {#third-column}
::: {style="text-align: center;font-size: 60%"}
![The cloud (Google drive)](images/Cloud.jpg){fig-alt="A serene view of the sky filled with soft, fluffy white clouds stretching across the horizon. The upper part of the image shows a gradient of blue, transitioning from deep to light as it meets the clouds." fig-align="center" width="100%"}
:::
::::
:::::

::: {style="text-align: left;font-size: 70%"}
Eventually, biomedical images (big data) can reach [terabytes or petabytes]{style="color:orange;"} in size, exceeding most standard file-sharing solutions.
:::

:::: {style="text-align: left;font-size: 80%"}
::: callout-tip
Effective image storage requires [infrastructure]{style="color:green;"}, optimization of [processing workflows]{style="color:green;"}, and [standardized]{style="color:green;"} sharing protocols. 
:::
::::

## Considerations for storage solutions{.center}
For selecting the storage modality, think that, as researchers, we do not want simply to store the dataset somewhere, we want to make it [accessible and usable]{style="color:green;"}. 

:::: {style="text-align: left;font-size: 80%"}
::: callout-tip
##  We want our images to be
- {{< bi search >}} Findable (Persistent identifiers, Indexed in a searchable resource).
- {{< bi hdd-fill >}} Accessible (Software requirements, Open file formats).
- {{< bi diagram-2-fill >}} Interoperable (rich metadata, standardized parameters).
- {{< bi recycle >}} Reusable (descriptive metadata, Clear license and usage rights).
:::
::::

## Where do I share my images?{.center}

::: {style="text-align: center;font-size: 80%"}
![Examples of generalist repositories](images/Repositories.png){fig-alt="A collection of logos representing various research data repositories, including OSF, Dryad, FRDR, DFDR, BioStudies, Zenodo, and Borealis. These repositories support open science by providing platforms for data sharing and preservation." width="100%"}
:::

## Features of some generalist repositories{.center}

::: {style="text-align: center;font-size: 60%"}
![From [Schmied. et al (2023)](https://www.nature.com/articles/s41592-023-01987-9/figures/7)](images/Repositories_Features.webp){fig-alt="Comparison table of various research data repositories including Zenodo, Figshare, Dryad, Bioimage Archive, Image Data Repository, Broad Bioimage Benchmark Collection, and The Cell Painting Gallery. The table details repository URLs, qualifications, support for non-image data, size limits, costs, and metadata requirements.  " width="100%"}
:::


## Specialized image repositories{.center}

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 60%"}
![[Image Data Resource (IDR)](https://idr.openmicroscopy.org/) and its [metadata scheme](https://github.com/IDR/idr-metadata)](images/Idr.png){fig-alt="Screenshot of the Image Data Resource (IDR) webpage, a public repository of bio-image datasets from research. The interface shows options to search for datasets, filter by type, and access specific studies. The image highlights statistics such as 132 studies, over 14 million images, and 411 TB of data, with a grid preview of available microscopy images." width="100%"}
:::
::::

:::: {#second-column}
::: {style="text-align: center;font-size: 60%"}
![[BioImage Archive](https://www.ebi.ac.uk/bioimage-archive/)](images/Bioimage.png){fig-alt="Screenshot of the BioImage Archive website, a free online repository for biological images. The page describes the archive's role in storing and distributing imaging data from various modalities, provided they are linked to a peer-reviewed publication or have significant value. It highlights submission policies, integration with other bioimaging databases like EMPIAR and IDR, and adherence to FAIR and REMBI guidelines. The right panel features additional resources, including a quick tutorial and a promotional banner." width="100%"}
:::
::::
:::::


## Other solutions

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 60%"}
![[OMERO](https://www.openmicroscopy.org/omero/)](images/Omero.png){fig-alt="Screenshot of the OMERO website homepage. The page features a dark-themed background with fluorescence microscopy images of cells. The OMERO logo is centered, followed by a description stating that OMERO is a secure central repository for managing, analyzing, and sharing microscopy images. It highlights compatibility with over 150 image file formats and supports access via desktop applications, web, or third-party software. A red button labeled 'Download Version 5.6' is prominently displayed." width="100%"}
:::
::::

:::: {#second-column}
::: {style="text-align: center;font-size: 60%"}
![[IRODS](https://irods.org/)](images/Irods.png){fig-alt="Screenshot of the iRODS website homepage. The page has a green background and presents iRODS as an open-source data management software. A circular diagram in the center highlights key iRODS capabilities, including Automated Ingest, Storage Tiering, Compliance, Publishing, Provenance, Integrity, Auditing, and Indexing. A 'Download iRODS' button is visible at the bottom." width="100%"}
:::
::::
:::::

::: {style="text-align: left;font-size: 80%"}
These instances are installed in dedicated network (core facility) space for long-term storage and sharing.
:::

## Characteristics of shared images{.center}

::: {style="text-align: left;font-size: 60%"}
- [Raw]{style="color:orange;"} and [processed]{style="color:green;"} images (processing operations should not alter the original image).
- [Uncompressed]{style="color:green;"} and [lossless]{style="color:green;"} open file formats.
- OME-TIFF retain original (i.e .czi) metadata.
- PNG (images with annotations) is preferred.
- Accurate and descriptive (machine readable) [naming conventions]{style="color:green;"} (i.e Subject_Group_Area_Marker): Use grouping factors to name images.
- Creation of [README files]{style="color:green;"} to contextualize and describe the content and methods used in the dataset. 
- Use CC-BY, or CC0 licenses.
:::

:::: {style="text-align: left;font-size: 80%"}
::: callout-tip
"We strongly discourage author statements that images â€˜are available upon requestâ€™, as this has been shown to be inefficient" ([Schmied et al. 2023](https://www.nature.com/articles/s41592-023-01987-9))
:::
::::

# Sharing reproducible workflows

## Image processing and analysis{.center}

In any research workflow, the analysis of images must be:

::::: {layout-ncol="3" layout-valign="center"}
:::: {#first-column}
### Objective
::: {style="text-align: center;font-size: 60%"}
![](images/Objective.png){fig-alt="Illustration of a book with a magnifying glass placed over it. The book has a yellow cover with a label on the top, and the magnifying glass has a red rim with a blue lens, symbolizing research, analysis, or study." fig-align="center" width="70%"}
:::
::::

:::: {#second-column}
### Reliable
::: {style="text-align: center;font-size: 60%"}
![From https://nexxis.com.au/](images/Realible.webp){fig-alt="Close-up of a metallic carabiner with the word 'RELIABILITY' engraved on it. The carabiner is securely fastened with a golden locking mechanism and is attached to a strong fabric strap." fig-align="center" width="100%"}
:::
::::

:::: {#third-column}
### Reproducible
::: {style="text-align: center;font-size: 60%"}
![From https://med.stanford.edu/](images/Replication.jpg){fig-alt="Illustration of a researcher in a white lab coat painting on a canvas, which contains an image of himself painting on a canvas in an infinite recursive loop. The background is red, creating a striking contrast. The artwork symbolizes scientific replication, self-referential processes, and the iterative nature of research. " fig-align="center" width="100%"}
:::
::::
:::::

## Image analysis from RDM perspective {.center}

:::: {style="text-align: left;font-size: 80%"}
From an RDM perspective, analysis of biomedical images ideally entails:

- {{< bi unlock >}} Access to large datasets.
- {{< bi file-earmark-text-fill >}} Records of image processing (code, scripts). 
- {{< bi share-fill >}} Sharing of results (images, tables, graphs). 
:::

:::: {style="text-align: left;font-size: 80%"}
::: callout-tip
Accurate, descriptive [naming conventions and README]{style="color:green;"} files with metadata, or codebooks are vital to assure the integrity of analysis pipelines. 
:::
::::

## Modular approach for image analysis{.center}

A [modular pipeline]{style="color:green;"} establishes the main image analysis tasks into independent sub tasks.

```{mermaid}
%%| fig-width: 20
%%| fig-height: 9

flowchart LR
A[Nuclei detection] --> B[Quantification] 
B --> C[Spatial analysis]
```

:::: {style="text-align: left;font-size: 80%"}
::: callout-tip
Modularity allows to [construct complex analysis pipelines]{style="color:green;"} from independent components that can function together. This promotes the [reuse]{style="text-decoration: underline;"} of independent modules.
:::
::::

## Hierarchy of image analysis tasks {.center}

::: {style="text-align: center;font-size: 60%"}
![From [Driscoll, M & Zaritsky, A. (2021)](https://doi.org/10.1242/jcs.254292)](images/Hierarchy.jpg){fig-alt="Diagram illustrating the hierarchy of image processing in biological research. The first section (A) differentiates between low-level signal-driven processes, such as filtering, segmentation, tracking, and registration, which generate outputs that are not directly interpretable, and high-level biology-driven processes like statistical analysis, inference, visualization, and categorization, which produce interpretable outputs. The second section (B) presents a workflow starting from image registration, followed by segmentation, tracking, filtering, and analysis, leading to visualization and interpretation." fig-align="center" width="50%"}
:::

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: left;font-size: 50%"}
**Low-level (technical knowledge)**

Transform images in other images or data:

- Preprocessing (e.g. deconvolution, noise removal)
- Object detection and segmentation (e.g. cells, intracellular organelles)
- Particle/object tracking 
:::
::::

:::: {#second-column}
::: {style="text-align: left;font-size: 50%"}
**High-level (disciplinary knowledge)**

Transform outputs from low-level tasks into information with biological meaning: 

- Data visualization 
- Fitting of statistical models 
- Statistical inference and uncertainty measurements
:::
::::
:::::


## Image analysis workflow {.center}

::: {style="text-align: center;font-size: 80%"}
![From  [Schmied et al., 2023](https://www.nature.com/articles/s41592-023-01987-9)](images/Graph_Workflows.webp){fig-alt="Diagram illustrating different image analysis workflows in biological research. The left panel shows microscopy images as input. The central panel presents three types of workflows: (a) an established workflow using predefined methods such as SynAct for synaptic activity analysis, (b) a new workflow composed of multiple processing steps like scripts, macros, pipelines, or plugins, and (c) a machine learning workflow utilizing neural networks for image classification. The right panel displays different outputs, including fluorescence intensity plots over time, segmented images, and classification counts." width="80%"}
:::

## Prioritize open/free software

There are dozens of open/free options to analyze research images and share reproducible workflows:

::: {style="text-align: left;font-size: 80%"}
- [Fiji](https://imagej.net/software/fiji/downloads)
- [CellPose](https://www.cellpose.org/)
- [Ilastik](https://www.ilastik.org/)
- [OMERO](https://www.openmicroscopy.org/omero/)
- [QuPath](https://qupath.github.io/) 
- [Python](https://www.python.org/)
- [Napari](https://napari.org/stable/)
- [Galaxy]( https://imaging.usegalaxy.eu/) or https://usegalaxy.eu/
:::

## BioImage as resource for images analysis pipelines{.center}

::: {style="text-align: left;font-size: 80%"}
[BioImage.IO](https://doi.org/10.1101/2022.06.07.495102) is a community-driven AI model repository that provides access to pretrained AI models with a plethora of open/free software partners. 
:::

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 80%"}
![[BioImage](https://bioimage.io/)](images/BioImage.io.png){fig-alt="Screenshot of the BioImage.IO website, showcasing the BioImage Model Zoo. The site offers access to advanced AI models for bioimage analysis with one-click integration. The page highlights compatibility with Fiji, ilastik, and ImJoy, and features a button labeled 'Explore the Zoo.' At the bottom, logos of community partners are displayed. The background contains a blue gradient with a stylized giraffe, trees, and animals." fig-align="center" width="100%"}
:::
::::

:::: {#Second-column}
::: {style="text-align: center;font-size: 80%"}
![From Bioimage](images/Bioimage_Examples.png){fig-alt="Screenshot of the BioImage Model Zoo, displaying multiple AI models for bioimage segmentation and analysis. Each model has a thumbnail, title, description, tags, and download count. Some models focus on nucleus, cell, and mitochondria segmentation using different deep learning techniques. The interface features icons for compatibility, licensing, and additional functionalities." fig-align="center" width="100%"}
:::
::::
:::::


## Image analysis workflows {.center}

::::: {layout-ncol="3" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 60%"}
![](images/CheckList_WorkflowsA.png){fig-align="center" width="100%"}
:::
::::

:::: {#second-column}
::: {style="text-align: center;font-size: 60%"}
![From  [Schmied et al., 2023](https://www.nature.com/articles/s41592-023-01987-9)](images/CheckList_WorkflowsB.png){fig-alt="Checklist for the publication of image-analysis workflows, divided into three categories: established workflows, new workflows, and machine learning workflows. Each category lists criteria such as citation of workflow or platform, inclusion of example data, manual region of interest (ROI), exact version, public availability, rationale, and limitations. The checklist includes recommendations for minimal, recommended, and ideal documentation standards.  " fig-align="center" width="100%"}
:::
::::

:::: {#third-column}
::: {style="text-align: center;font-size: 60%"}
![](images/CheckList_WorkflowsC.png){fig-align="center" width="100%"}
:::
::::
:::::

## Why to use code?{.center}

::: {style="text-align: left;font-size: 80%"}
"The mouse is antisocial. The GUI is antisocial. So, what's that mean? You have a problem to solve, and you solve it with a GUI. What do you have? A problem solved. But when you solve it with a command line interface in a scripting environment, you have an [artifact]{style="color:green;"}. And all of the sudden, that artifact can be shared with someone" **Jeffrey Snover**
:::

::: {style="text-align: center;font-size: 150%"}
Use [CODE]{style="color:green;"} not the [mouse]{style="color:red;"}!
:::


## Overall{.center}

::: callout-tip
- Keep track of all changes and analysis procedures perform to images. 
- Organize and link files (using naming conventions) through processing to avoid errors.
- Select final or intermediate results to share considering aspects such as storing space and long-term preservation. 
:::


## Publication of analysis results{.center}

:::: {style="text-align: left;font-size: 80%"}
Sharing of [research objects]{style="color:green;"} in public, active research management repositories, like the [Open Science Framework (OSF)](https://osf.io/) is an excellent strategy to promote open, reproducible research. Please consider sharing illustrative images, figures and tables used in publication.   
::::

::: callout-tip
In general, these are lower resolution images/figures (.png) not used for analysis but for illustration in research reports (thesis, articles).
:::

## Examples of results images

::::: {layout-ncol="2" layout-valign="center"}
:::: {#first-column}
::: {style="text-align: center;font-size: 60%"}
![From  [Schmied et al., 2023](https://www.nature.com/articles/s41592-023-01987-9)](images/CheckList_Formatting.webp){fig-alt="Guidelines for research image presentation, illustrating best practices for clarity and accuracy. Sections include: (a) focusing on relevant image content through cropping and resizing, (b) separating individual images, (c) indicating the origin of insets, (d) adjusting brightness and contrast appropriately, (e) ensuring channel annotations are visible against the background, (f) using grayscale channels to improve visibility of features, and (g) explaining channel values with calibration scales and indicators. " width="100%"}
:::
::::

:::: {#second-column}
::: {style="text-align: left;font-size: 80%"}
Click to see examples of published results [images](https://osf.io/mb257/) and [figures](https://osf.io/jxb4z/).
:::
::::
:::::

## Data should be shared{.center}

:::: {style="text-align: left;font-size: 80%"}
â€˜Data is available on requestâ€™ statements in publications are found to be often unreliable in practice ([Schmidt et al., 2024](https://onlinelibrary.wiley.com/doi/10.1111/jmi.13317)) 
::::

::: callout-tip
## Open Science Principle
Share the data as openly as possible and keep it only as closed as necessary. 
:::


## {{< bi share-fill >}} Sharing data is a professional responsibility{.center}

Depositing a dataset in a repository is [NOT ONLY]{style="color:red;"} an exercise in meeting the requirements of funding agencies and journals. It is an [ethical and professional responsibility]{style="color:green;"} of researchers to ensure reproducible research, and the access and reuse of research data.

## Therefore, research needs to move towards{.center} 

- [Competent]{style="color:green;"} researchers in [RDM]{style="text-decoration: underline;"} and data analysis.
- [Standardized approaches]{style="color:green;"} to [sharing]{style="text-decoration: underline;"} raw data and analysis code to support research findings.
- Researchers with a [commitment to transparency]{style="color:green;"} and best research practices to ensure [integrity]{style="text-decoration: underline;"}.


## {.center}

::: {style="text-align: center;font-size: 100%"}
![Visit [FRDR](https://www.frdr-dfdr.ca/repo/) or [Borealis](https://borealisdata.ca/)](images/Repositories_Logo.png){fig-alt="Logos of two Canadian research data repositories: FRDR (Federated Research Data Repository) and Borealis. The FRDR logo features a geometric pattern of yellow squares forming a diamond shape, with the repository name in black and gold text. The Borealis logo includes an artistic depiction of the Northern Lights over mountains and a lake, with the repository name in bold white text. " fig-align="center" width="1000" height="220"}
:::


------------------------------------------------------------------------

## Resources and support {.smaller}

::::: {layout-ncol="2"}
:::: {#first-column}
::: {style="text-align: center;font-size: 80%"}
### Supporting material

- [FAIR image data](https://www.eurobioimaging.eu/data-services/fair-image-data/) 
- [FAIR Bioimage Metadata](https://training.galaxyproject.org/training-material/topics/imaging/tutorials/bioimage-metadata/tutorial.html)
- [FRDR documentation](https://www.frdr-dfdr.ca/docs/en/depositing_data/)
- [Training resources](https://alliancecan.ca/en/services/research-data-management/learning-and-training/training-resources) from the Alliance 
:::
::::

:::: {#second-column}
::: {style="text-align: center;font-size: 80%"}
![This presentation is available [here](https://github.com/Alliance-RDM-GDR/RDM_BioImageFAIR) (English or French)](images/QR-code.png){fig-alt="A QR code image that redirects to the presentation located in a GitHub repository." fig-align="center" width="200" height="200"}
:::
::::
:::::

### Support Services:

::: {style="text-align: left;font-size: 90%"}
Contact us to ensure that your data is well prepared and can be effectively shared with the research community.

-   Email: rdm-gdr\@alliancecan.ca
-   https://www.frdr-dfdr.ca/repo/
:::