<!DOCTYPE html>
<html lang="en"><head>
<script src="RDM_BioImageFAIR_en_files/libs/clipboard/clipboard.min.js"></script>
<script src="RDM_BioImageFAIR_en_files/libs/quarto-html/tabby.min.js"></script>
<script src="RDM_BioImageFAIR_en_files/libs/quarto-html/popper.min.js"></script>
<script src="RDM_BioImageFAIR_en_files/libs/quarto-html/tippy.umd.min.js"></script>
<link href="RDM_BioImageFAIR_en_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="RDM_BioImageFAIR_en_files/libs/quarto-html/light-border.css" rel="stylesheet">
<link href="RDM_BioImageFAIR_en_files/libs/quarto-html/quarto-syntax-highlighting-e26003cea8cd680ca0c55a263523d882.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<link href="RDM_BioImageFAIR_en_files/libs/quarto-contrib/reveal-auto-agenda-0.0.3/reveal-auto-agenda.css" rel="stylesheet">
<link href="RDM_BioImageFAIR_en_files/libs/quarto-contrib/bootstrap-icons-1.11.1/all.css" rel="stylesheet"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.6.39">

  <meta name="author" content="FRDR curation team">
  <title>Handling biomedical images and sharing reproducible workflows</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="RDM_BioImageFAIR_en_files/libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="RDM_BioImageFAIR_en_files/libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
  </style>
  <link rel="stylesheet" href="RDM_BioImageFAIR_en_files/libs/revealjs/dist/theme/quarto-bbe7401fe57d4b791b917637bb662036.css">
  <link rel="stylesheet" href="styles.css">
  <link href="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
  <meta name="mermaid-theme" content="forest">
  <script src="RDM_BioImageFAIR_en_files/libs/quarto-diagram/mermaid.min.js"></script>
  <script src="RDM_BioImageFAIR_en_files/libs/quarto-diagram/mermaid-init.js"></script>
  <link href="RDM_BioImageFAIR_en_files/libs/quarto-diagram/mermaid.css" rel="stylesheet">
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">Handling biomedical images and sharing reproducible workflows</h1>
  <p class="subtitle">A primer for researchers</p>

<div class="quarto-title-authors">
<div class="quarto-title-author">
<div class="quarto-title-author-name">
FRDR curation team 
</div>
<div class="quarto-title-author-email">
<a href="mailto:curators@frdr-dfdr.ca">curators@frdr-dfdr.ca</a>
</div>
        <p class="quarto-title-affiliation">
            Digital Research Alliance of Canada
          </p>
    </div>
</div>

</section>
<section>
<section id="understanding-bioimaging" class="title-slide slide level1 agenda-slide center">
<h1>Understanding BioImaging</h1>
<div class="agenda-heading">
<p>Agenda</p>
</div>
<div class="agenda">
<ol>
<li><div class="agenda-active">
<p>Understanding BioImaging</p>
</div></li>
<li><div class="agenda-inactive agenda-post-active">
<p>Planning and adquiring images</p>
</div></li>
<li><div class="agenda-inactive agenda-post-active">
<p>Gathering image metadata</p>
</div></li>
<li><div class="agenda-inactive agenda-post-active">
<p>Organizing and sharing image data</p>
</div></li>
<li><div class="agenda-inactive agenda-post-active">
<p>Sharing reproducible workflows</p>
</div></li>
</ol>
</div>
</section>
<section id="section" class="slide level2">
<h2></h2>
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/ImagingSamples.jpg" class="quarto-figure quarto-figure-center" style="width:100.0%" alt="A diagram illustrating different types of biological imaging techniques and their applications. The top section, labeled 'Sample Imaging,' includes icons representing various research fields such as molecular biology, genetics, microbiology, and anatomy. Below, three imaging modalities are depicted: 'Electron microscopy' (in light blue), 'Light microscopy' (in dark blue), and 'Human bioimaging' (in dark red), indicating their areas of application. At the bottom, 'Image data analysis' is shown, emphasizing the computational aspect of image processing. The diagram highlights the role of imaging across multiple research disciplines."></p>
</figure>
</div>
</div>
</section>
<section id="images-contain-countless-data" class="slide level2 center">
<h2>Images contain countless data</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/DataTheraphy.png" style="width:65.0%" alt="A black-and-white cartoon illustration depicting a therapy session. A patient is lying on a couch, expressing their concerns to a therapist who is taking notes. The patient has a thought bubble showing a massive wave labeled 'DATA' about to crash over a small, helpless figure. The cartoon humorously represents the overwhelming feeling of dealing with large amounts of data, a common challenge in research and data science. The drawing is signed by Henning Falk."></p>
<figcaption>Henning Falk (2022) - NumFOCUS</figcaption>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: left;font-size: 80%">
<p>Bioimaging has entered the realm of big data comprising <span style="color:green;">increasingly complex datasets</span>. We face numerous challenges, specifically, proper <span style="text-decoration: underline;">data handling</span> and <span style="text-decoration: underline;">management</span> and the creation and sharing of <span style="text-decoration: underline;">reproducible image analysis</span> workflows.</p>
</div>
</div>
</div>
</div>
</section>
<section id="bioimage-life-cicle" class="slide level2">
<h2>BioImage Life cicle</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/ImageLifeCycle.jpg" style="width:80.0%" alt="A circular diagram illustrating the bioimaging research data management (RDM) lifecycle, titled 'Core Facility: Infrastructure and Bioimaging RDM Support.' At the center, a researcher icon represents the user interacting with different stages of the workflow. The lifecycle is divided into segments: 'Image Acquisition' (purple), 'Image File Storage &amp; Access' (yellow), 'Processing &amp; Bioimage Analysis' (green), '(Data-) Publication' (blue), 'Archiving, Long-Term Storage' (light gray), and 'Planning, Data Search &amp; Re-use' (gray). Around the outer circle, key supporting infrastructure elements are represented, including cloud storage, file system storage, efficient data transfer, formats, standards, tools, permission systems, data security, web access, connectivity, and data integration. The diagram highlights the structured approach to managing and supporting bioimaging research data."></p>
<figcaption>BioImage life cycle from <a href="https://doi.org/10.1111/jmi.13317">Schmidt, et al.&nbsp;2024</a></figcaption>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: left;font-size: 90%">
<p>BioImages have the potential for scientific discovery beyond its original acquisition purpose when handled according to <span style="color:green;">FAIR principles</span> (<a href="https://onlinelibrary.wiley.com/doi/10.1111/jmi.13317">Schmidt, et al.&nbsp;2024</a>; see <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC4792175/">Wilkinson, et al.&nbsp;2016</a>).</p>
</div>
</div>
</div>
</div>
</section></section>
<section>
<section id="planning-and-adquiring-images" class="title-slide slide level1 agenda-slide center">
<h1>Planning and adquiring images</h1>
<div class="agenda-heading">
<p>Agenda</p>
</div>
<div class="agenda">
<ol>
<li><div class="agenda-inactive agenda-pre-active">
<p>Understanding BioImaging</p>
</div></li>
<li><div class="agenda-active">
<p>Planning and adquiring images</p>
</div></li>
<li><div class="agenda-inactive agenda-post-active">
<p>Gathering image metadata</p>
</div></li>
<li><div class="agenda-inactive agenda-post-active">
<p>Organizing and sharing image data</p>
</div></li>
<li><div class="agenda-inactive agenda-post-active">
<p>Sharing reproducible workflows</p>
</div></li>
</ol>
</div>
</section>
<section id="overview" class="slide level2 center">
<h2>Overview</h2>
<div style="text-align: left;font-size: 80%">
<p>Conventional file-system-based storage is quickly reaching its <span style="color:red;">limits</span>. Before the data are generated, researchers must consider how it will be <span style="text-decoration: underline;">stored</span>, <span style="text-decoration: underline;">moved</span>, <span style="text-decoration: underline;">documented</span> and <span style="text-decoration: underline;">analyzed</span> during (and after) the project’s lifetime.</p>
</div>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Overcapacity.png" class="quarto-figure quarto-figure-center" style="width:80.0%" alt="A screenshot of the Windows File Explorer 'This PC' section, displaying storage devices and their available disk space. The image highlights two key areas: (1) the 'This PC' navigation option on the left panel, and (2) the 'Devices and drives' section, showing multiple hard drives and partitions, some nearly full with red-colored storage bars. This image illustrates disk space management challenges and overcapacity warnings in Windows."></p>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: left;font-size: 60%">
<p><strong>Current directions:</strong></p>
<ul>
<li>Leverage on research data management plans (<span style="color:green;">DMP</span>)</li>
<li>Cloud-based storage and platforms (i.e OMERO) will become a <span style="color:green;">MUST</span>.</li>
<li>Need for standardized formats (<span style="color:green;">NGFFs</span>).</li>
</ul>
</div>
</div>
</div>
</div>
</section>
<section id="planning-imaging-experiments" class="slide level2 center">
<h2>Planning imaging experiments</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<h3 id="imaging-hadware"><i class="bi-gear-fill " style="" role="img" aria-hidden="true"></i> Imaging Hadware</h3>
<div style="text-align: left;font-size: 80%">
<ul>
<li>Imaging device (light, confocal, electron).</li>
<li>Define imaging parameters - suitable for image analysis.</li>
</ul>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<h3 id="storage-of-images"><i class="bi-database-fill " style="" role="img" aria-hidden="true"></i> storage of images</h3>
<div style="text-align: left;font-size: 80%">
<ul>
<li>Image size - Gigas or terabytes.</li>
<li>Image complexity - Single or multiple focal planes, Live or time-lapse images.</li>
</ul>
</div>
</div>
</div>
</div>
<div style="text-align: left;font-size: 80%">
<div class="callout callout-warning callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>How will you handle your images?</strong></p>
</div>
<div class="callout-content">
<p>While panning an imaging experiment, consider the most <span style="text-decoration: underline;">suitable approach</span> given your capabilities for <span style="color:green;">data handling and analysis</span>.</p>
</div>
</div>
</div>
</div>
</section>
<section id="data-adquisition" class="slide level2 center">
<h2>Data adquisition</h2>
<div style="text-align: left;font-size: 80%">
<ul>
<li><i class="bi-gear-fill " style="" role="img" aria-hidden="true"></i> Desirable instrument <span style="color:green;">setup and calibration</span> and <span style="color:green;">imaging parameters</span> (size, bit depth, saturation, etc.).</li>
<li><i class="bi-card-image " style="" role="img" aria-hidden="true"></i> Prioritize <span style="color:green;">open file formats</span> (.TIFF) over proprietary (.CZI).</li>
<li><i class="bi-database-fill " style="" role="img" aria-hidden="true"></i> Recording of <span style="color:green;">metadata</span>.</li>
</ul>
</div>
<div style="text-align: left;font-size: 80%">
<div class="callout callout-tip callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Adquisition parameters are key</strong></p>
</div>
<div class="callout-content">
<p>During imaging it is necessary to consider a compromise between the <span style="color:green;">parameters</span> needed to answer the research question (magnification, size, bit depth) and the <span style="color:green;">available processing power</span> (storage, computing power).</p>
</div>
</div>
</div>
</div>
</section>
<section id="what-parameters-should-i-use-to-answer-my-research-question" class="slide level2 center">
<h2>What parameters should I use to answer my research question?</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Image_8bit.png" style="width:100.0%" alt="A fluorescence microscopy image of a coronal mouse brain section displayed in an image viewer. The image is captured from a `.czi` file and shown as an 8-bit TIFF format with a size of 23MB. The fluorescence signal appears in green, highlighting specific structures within the brain tissue. The metadata in the image viewer indicates dimensions of 9086.62 x 7290.43 microns, corresponding to a resolution of 2752 x 2208 pixels. This type of imaging is commonly used in neuroscience research to visualize protein expression, cellular structures, or anatomical regions."></p>
<figcaption>8 bit image - Courtesy of Daniel Manrique-Castano</figcaption>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Image_16bit.png" style="width:100.0%" alt="The image shows the same as the left one for a 16-bit depth."></p>
<figcaption>16 bit image - Courtesy of Daniel Manrique-Castano</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="consider-that" class="slide level2 center">
<h2>Consider that…</h2>
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Microscope.jpeg" class="quarto-figure quarto-figure-center" style="width:100.0%" alt="A humorous research-themed illustration combining text and a drawing of a microscope. On the left side, bold black text reads 'BRO, DO YOU EVEN SCOPE?' with an illustration of a compound microscope below it. On the right side, within a light blue background, the text emphasizes the importance of training in microscopy: 'A microscope is as good as the person that operates it. Experiment quality can be boosted with training.' The words 'microscope' and 'Experiment quality' are highlighted in blue. The image humorously promotes proper training for obtaining high-quality research results using microscopes."></p>
</figure>
</div>
</div>
</section>
<section id="transforming-file-formats" class="slide level2 center">
<h2>Transforming file formats</h2>
<div style="text-align: left;font-size: 80%">
<p>After acquiring original images in <span style="color:red;">proprietary formats</span> (i.e .CZI or .LIF), researchers can use different tools to open and transform the images to <span style="color:green;">open formats</span> (.TIFF).</p>
</div>
<div>

</div>
<div class="quarto-layout-panel" data-layout-nrow="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-row" class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 70%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Bioformats-Omero.png" style="width:70.0%" alt="A screenshot of the Bio-Formats 8.1.0 download page, a software tool for reading and converting life sciences image file formats. The background contains a fluorescence microscopy image of cells stained with blue, green, and yellow markers. The main heading, 'Bio-Formats 8.1.0 Downloads,' is displayed in large white text. Bio-Formats is an essential tool for researchers working with multidimensional microscopy data."></p>
<figcaption>Bio-Formats from <a href="https://www.openmicroscopy.org/bio-formats/downloads/">Open Microscopy Environment</a> (compatible with <a href="https://imagej.net/formats/bio-formats">FIJI</a>)</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="second-row" class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 70%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Bioformats-python.png" style="width:70.0%" alt="A screenshot of the 'python-bioformats 4.1.0' package release page. The page has a blue background and prominently displays the package name and version. Below, a command snippet shows how to install the package using pip: 'pip install python-bioformats,' with a copy button next to it. This package enables Python users to work with Bio-Formats, a library for reading and processing research image data."></p>
<figcaption>Bio-Formats for <a href="https://pypi.org/project/python-bioformats/">Python</a></figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="the-ome.tiff-format" class="slide level2 center">
<h2>The OME.TIFF format</h2>
<div style="text-align: left;font-size: 80%">
<p>Is a format developed by the <a href="https://www.openmicroscopy.org/">Open Microscopy Environment (OME)</a> based on TIFF specification. OME-TIFF incorporates:</p>
<ul>
<li><a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2005-6-5-r47">OME-XML</a> metadata.</li>
<li>Support multidimensional, multi image data (see <a href="https://downloads.openmicroscopy.org/images/OME-TIFF/2015-01/bioformats-artificial/">examples</a>).</li>
<li>Supports complex multidimensional and high-content screening data using <a href="http://bigtiff.org/">BigTiff</a>, given the limitations of TIFF (up to 4GB).</li>
<li>Can distribute the data across multiple files (see <a href="https://downloads.openmicroscopy.org/images/OME-TIFF/2015-01/tubhiswt-4D/">example</a>).</li>
</ul>
</div>
</section>
<section id="next-generation-file-format-ome-zarr" class="slide level2 center">
<h2>Next-Generation File Format OME-Zarr</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/ImagePyramid.jpeg" style="width:100.0%" alt="A schematic illustration of an image pyramid, a concept used in image processing and bioimaging. The pyramid represents different resolution levels of the same image, with a pink-stained brain tissue sample shown at each level. The base of the pyramid represents 'Full resolution, Full area,' while the middle and upper levels correspond to '1/2 resolution, 1/4 area' and '1/4 resolution, 1/16 area,' respectively. This visualization highlights how images are stored at multiple scales to optimize processing and analysis in whole-slide imaging and multi-resolution microscopy."></p>
<figcaption>Image Pyramid</figcaption>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: left;font-size: 70%">
<ul>
<li>Efficient storage and display of the image (Efficient slicing at arbitrary angles).</li>
<li>Compatible with <a href="https://github.com/saalfeldlab/n5-viewer">Fiji</a>, <a href="https://github.com/ome/napari-ome-zarr">Napari</a>, <a href="https://kitware.github.io/itk-vtk-viewer/docs/index.html">ITK/VTK Viewer</a>, Google’s <a href="https://github.com/google/neuroglancer?tab=readme-ov-file">Neuroglancer</a>, <a href="https://github.com/hms-dbmi/vizarr">Vizarr</a>, and Allen Institute’s <a href="https://www.allencell.org/3d-cell-viewer.html">3D Cell Viewer</a>.</li>
<li>For Conda-based batch conversion to OME-Zarr visit <a href="https://github.com/Euro-BioImaging/BatchConvert">this repository</a>.</li>
<li>Handle <a href="(https://link.springer.com/article/10.1007/s00418-023-02209-1)">OME-Zarr</a> datasets in Python using <a href="https://github.com/Euro-BioImaging/ome_zarr_pyramid">this tool</a>.</li>
</ul>
</div>
</div>
</div>
</div>
</section>
<section id="take-home-message" class="slide level2 center">
<h2>Take home message</h2>
<div style="text-align: left;font-size: 100%">
<p>Strategic <span style="color:green;">planing and data management</span>, cloud-ready <span style="color:green;">platforms and formats</span>, and institutional support ensure scalable, accessible, and <span style="text-decoration: underline;">reusable</span> imaging data.</p>
</div>
</section></section>
<section>
<section id="gathering-image-metadata" class="title-slide slide level1 agenda-slide center">
<h1>Gathering image metadata</h1>
<div class="agenda-heading">
<p>Agenda</p>
</div>
<div class="agenda">
<ol>
<li><div class="agenda-inactive agenda-pre-active">
<p>Understanding BioImaging</p>
</div></li>
<li><div class="agenda-inactive agenda-pre-active">
<p>Planning and adquiring images</p>
</div></li>
<li><div class="agenda-active">
<p>Gathering image metadata</p>
</div></li>
<li><div class="agenda-inactive agenda-post-active">
<p>Organizing and sharing image data</p>
</div></li>
<li><div class="agenda-inactive agenda-post-active">
<p>Sharing reproducible workflows</p>
</div></li>
</ol>
</div>
</section>
<section id="overview-1" class="slide level2">
<h2>Overview</h2>
<p><span style="color:green;">High-quality metadata</span> is crucial for making imaging data <strong>FAIR</strong>.</p>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Fragmented.png" style="width:80.0%" alt="A vibrant abstract digital artwork featuring a fragmented human face interwoven with geometric shapes, colorful textures, and technological elements. The composition includes a large, expressive eye surrounded by streaks of color, circuits, and intersecting lines. Additional elements, such as a stylized mouth and curved, ribbon-like structures, contribute to the surreal aesthetic."></p>
<figcaption>Image from <a href="https://www.deviantart.com/bonsaiartistry/art/Fragmented-Whispers-Abstract-Collage-Rhapsody-978347147">DevianArt</a></figcaption>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: left;font-size: 80%">
<p><span style="color:red;">Current challenges:</span></p>
<ul>
<li>A fragmented ecosystem <span style="color:red;">lacking clear metadata standards</span> that leads to inconsistent documentation.</li>
<li>Journals and institutions lacking uniform guidelines.</li>
</ul>
</div>
</div>
</div>
</div>
</section>
<section id="recommended-metadata-for-biological-images-rembi" class="slide level2 center">
<h2>Recommended Metadata for Biological Images (REMBI)</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Rembi.webp" style="width:70.0%" alt="A Venn diagram illustrating the overlapping metadata needs of different research domains. The three primary categories are 'Research biologist’s needs' (blue), 'Computer vision researcher’s needs' (green), and 'Imaging scientist’s needs' (yellow). Their intersection highlights the 'Recommended metadata,' which serves as a standardized set of information beneficial across these disciplines."></p>
<figcaption>Image from <a href="https://doi.org/10.1038/s41592-021-01166-8">Sarkans, et al.&nbsp;2021</a></figcaption>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: left;font-size: 80%">
<p><a href="https://www.nature.com/articles/s41592-021-01166-8">REMBI</a> advice metadata guidelines to address the needs of diverse biomedical imaging communities.</p>
<p>A summary template is available <a href="https://github.com/Alliance-RDM-GDR/RDM_BioimageFAIR/blob/main/resources/REMBI_Overview.xlsx">here</a>.</p>
</div>
</div>
</div>
</div>
</section>
<section id="section-1" class="slide level2">
<h2></h2>
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Rembi.png" style="width:60.0%" alt="A structured table outlining the different metadata modules required for standardized bioimaging data documentation. The table consists of several sections: 'Study,' 'Study Component,' 'Biosample,' 'Specimen,' 'Image Acquisition,' 'Image Data,' 'Image Correlation,' and 'Analyzed Data.' Each row specifies an attribute (e.g., study type, imaging method, biological entity), a description of its significance, the data entry method (text, ontology, extracted data), and related ontologies (such as EDAM-BIOIMAGING, FBbi, OME, and EFO). This table highlights essential metadata elements needed to ensure reproducibility, interoperability, and structured data sharing in bioimaging research."></p>
<figcaption><a href="https://github.com/Alliance-RDM-GDR/RDM_BioimageFAIR/blob/main/resources/REMBI_Overview.xlsx">Download</a> the template</figcaption>
</figure>
</div>
</div>
</section>
<section id="use-controlled-vocabulary" class="slide level2 center">
<h2>Use controlled vocabulary</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<p>Use <span style="color:green;">controlled vocabulary</span> (ontologies) to specify objects, their categories and relationships.</p>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<div style="text-align: left; font-size: 80%">
<table class="caption-top">
<thead>
<tr class="header">
<th style="text-align: left;"><strong>Name</strong></th>
<th style="text-align: left;"><strong>Ontology</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Organisms</td>
<td style="text-align: left;"><a href="Widely%20supported,%20non-proprietary">NCBI</a></td>
</tr>
<tr class="even">
<td style="text-align: left;">Genes</td>
<td style="text-align: left;"><a href="https://www.ncbi.nlm.nih.gov/gene/">NCBI</a></td>
</tr>
<tr class="odd">
<td style="text-align: left;">Proteins</td>
<td style="text-align: left;"><a href="https://www.uniprot.org/">Uniprot</a></td>
</tr>
<tr class="even">
<td style="text-align: left;">Imaging Methods</td>
<td style="text-align: left;"><a href="https://www.ebi.ac.uk/ols4/ontologies/fbbi">FBBI</a></td>
</tr>
<tr class="odd">
<td style="text-align: left;">Exp. factors</td>
<td style="text-align: left;"><a href="https://www.ebi.ac.uk/ols4/ontologies/efo">EFO</a></td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</section>
<section id="web-based-ontology-services" class="slide level2 center">
<h2>Web-based ontology services</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/OLS.png" style="width:100.0%" alt="Screenshot of the EMBL-EBI Ontology Lookup Service (OLS) homepage. The interface provides a search bar for finding biomedical ontology terms, with options to filter by exact match, include obsolete terms, and include imported terms. Below, there are three sections: 'About OLS,' describing its function as a repository for biomedical ontologies; 'Related Tools,' mentioning additional ontology mapping services like OxO and ZOOMA; and 'Report an Issue,' directing users to a GitHub issue tracker for feedback and ontology requests."></p>
<figcaption><a href="https://www.ebi.ac.uk/ols4/">EMBL-EBI Ontology Lookup Service</a></figcaption>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Bioportal.png" style="width:100.0%" alt="Screenshot of Bioportal. This is a comprehensive repository of biomedical ontologies. The interface provides a search bar for finding biomedical ontology terms, with options for advanced search. It shows also a graphs depicting visits and key site statistics."></p>
<figcaption><a href="https://bioportal.bioontology.org/">Biportal</a></figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="record-metadata-from-imaging-devices" class="slide level2 center">
<h2>Record metadata from imaging devices</h2>
<div style="text-align: center;font-size: 90%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/MicroMetaApp.png" style="width:70.0%" alt="Screenshot of the Micro Meta App, a microscopy metadata management tool. The top section displays the app's banner with the title 'MICRO META APP' and the tagline 'Microscopy Metadata for the Real World.' Below, two main sections allow users to 'Manage Instrument' and 'Manage Settings,' providing options to document hardware components and acquisition settings. The lower half of the image shows detailed interface views, including labeled microscope components such as objectives, cameras, filter sets, and laser settings, highlighting metadata fields like manufacturer, gain, magnification, and bit depth."></p>
<figcaption>MicroMeta App</figcaption>
</figure>
</div>
</div>
<div style="text-align: left;font-size: 80%">
<p>Visit the <a href="https://wu-bimac.github.io/MicroMetaApp.github.io/">MicroMeta App</a> and the associated <a href="https://www.nature.com/articles/s41592-021-01315-z">research article</a></p>
</div>
</section>
<section id="methods-j2-plugin" class="slide level2 center">
<h2>Methods J2 PlugIn</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/MethodsJ2.png" style="width:100.0%" alt="Screenshot of a text document generated using the MethodsJ2 tool based on user input and a Micro-Meta App hardware file. The document details the imaging setup, including a Zeiss Axiovert 200M inverted microscope configured for Widefield Epifluorescence microscopy, controlled with Zen software. The text specifies the optical components, imaging parameters such as voxel size (0.14 μm), and details of fluorescence excitation using an X-Cite 120 LED light source with various excitation/emission filters and dichroic mirrors. The acknowledgments section credits the Advanced BioImaging Facility (ABIF) at McGill and Joel Ryan for assistance."></p>
<figcaption>Example from MethodsJ2 <a href="https://github.com/ABIF-McGill/MethodsJ2?tab=readme-ov-file">repository</a></figcaption>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: left;font-size: 80%">
<p>The MethodsJ2 Fiji plugIn <a href="%7Bstyle=%22color:green;%22%7D">generates text</a> for microscopy <span style="text-decoration: underline;">materials and methods</span> by extracting information from metadata (MicroMeta App file). Visit here the <a href="https://github.com/ABIF-McGill/MethodsJ2?tab=readme-ov-file">GitHub repository</a> or the associated <a href="https://www.nature.com/articles/s41592-021-01290-5">research article</a>.</p>
</div>
</div>
</div>
</div>
</section>
<section id="omero-one-of-the-best-choices" class="slide level2 center">
<h2>OMERO: One of the best choices</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Omero_Metadata.jpg" style="width:100.0%" alt="Diagram illustrating the workflow of metadata handling in OMERO, from image acquisition on a microscope to metadata import and annotation. The process involves vendor image containers being imported via the OMERO importer, where metadata is extracted using Bio-Formats. Metadata is structured following facility specifications and ontology standards. The OMERO.mde system organizes metadata into structured key-value pairs, enabling visualization, annotation, and export in formats such as .txt, .tsv, and .csv for reuse in IDR format."></p>
<figcaption>From <a href="https://www.nature.com/articles/s41592-021-01288-z">Kunis et al., 2021</a></figcaption>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: left;font-size: 80%">
<p>OMERO incorporates <a href="https://www.nature.com/articles/s41592-021-01288-z">MDEmic</a> (MetaData Editor for microscopy) an tool that provides an easy way to explore and edit metadata from images.</p>
</div>
</div>
</div>
</div>
</section>
<section id="metadata-to-consider" class="slide level2 center">
<h2>Metadata to consider</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/CheckList_ImagesA.png" style="width:90.0%" alt="Checklist for image publishing, covering aspects of image format and image colors/channels. The checklist includes requirements for focusing on relevant content (cropping, rotation, resizing), separating individual images, and providing example images for quantification. It also emphasizes annotation visibility, brightness/contrast adjustments, consistent image comparisons, and color accessibility for color-blind individuals. Recommended steps include adding grayscale versions, intensity scales, and adjustments like gamma correction and pseudocoloring."></p>
<figcaption>From <a href="https://www.nature.com/articles/s41592-023-01987-9">Schmied et al., 2023</a></figcaption>
</figure>
</div>
</div>
</div>
<div id="Second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/CheckList_ImagesB.png" style="width:90.0%"></p>
<figcaption>From <a href="https://www.nature.com/articles/s41592-023-01987-9">Schmied et al., 2023</a></figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="we-need-to-move-toward" class="slide level2 center">
<h2>We need to move toward:</h2>
<ul>
<li>Better consolidation of metadata <span style="color:green;">standards</span>.</li>
<li>Development of <span style="color:green;">tools</span> for automated metadata collection (includes collaboration with microscope manufacturers).</li>
<li>Research infrastructures and BioImaging support for proper <span style="color:green;">metadata recording</span>.</li>
</ul>
</section></section>
<section>
<section id="organizing-and-sharing-image-data" class="title-slide slide level1 agenda-slide center">
<h1>Organizing and sharing image data</h1>
<div class="agenda-heading">
<p>Agenda</p>
</div>
<div class="agenda">
<ol>
<li><div class="agenda-inactive agenda-pre-active">
<p>Understanding BioImaging</p>
</div></li>
<li><div class="agenda-inactive agenda-pre-active">
<p>Planning and adquiring images</p>
</div></li>
<li><div class="agenda-inactive agenda-pre-active">
<p>Gathering image metadata</p>
</div></li>
<li><div class="agenda-active">
<p>Organizing and sharing image data</p>
</div></li>
<li><div class="agenda-inactive agenda-post-active">
<p>Sharing reproducible workflows</p>
</div></li>
</ol>
</div>
</section>
<section id="images-could-be-in-many-places" class="slide level2 center">
<h2>Images could be in many places</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="3">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Laptop.jpg" style="width:100.0%" alt="A laptop with a black screen placed on a wooden table in a café setting. A cup of coffee on a saucer is next to it. The background has a wire mesh window with plants visible outside. "></p>
<figcaption>Laptop of students and postdocs</figcaption>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Network.jpg" style="width:100.0%" alt="A close-up of a network switch with multiple Ethernet cables plugged in. The cables are orange and blue, neatly organized and connected to the switch. A label with the text 'NorthC' is visible on the top right corner of the device."></p>
<figcaption>Institute network</figcaption>
</figure>
</div>
</div>
</div>
<div id="third-column" class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Cloud.jpg" style="width:100.0%" alt="A serene view of the sky filled with soft, fluffy white clouds stretching across the horizon. The upper part of the image shows a gradient of blue, transitioning from deep to light as it meets the clouds."></p>
<figcaption>The cloud (Google drive)</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
<div style="text-align: left;font-size: 70%">
<p>Eventually, biomedical images (big data) can reach <span style="color:orange;">terabytes or petabytes</span> in size, exceeding most standard file-sharing solutions.</p>
</div>
<div style="text-align: left;font-size: 80%">
<div class="callout callout-tip callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Tip</strong></p>
</div>
<div class="callout-content">
<p>Effective image storage requires <span style="color:green;">infrastructure</span>, optimization of <span style="color:green;">processing workflows</span>, and <span style="color:green;">standardized</span> sharing protocols.</p>
</div>
</div>
</div>
</div>
</section>
<section id="considerations-for-storage-solutions" class="slide level2 center">
<h2>Considerations for storage solutions</h2>
<p>For selecting the storage modality, think that, as researchers, we do not want simply to store the dataset somewhere, we want to make it <span style="color:green;">accessible and usable</span>.</p>
<div style="text-align: left;font-size: 80%">
<div class="callout callout-tip callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>We want our images to be</strong></p>
</div>
<div class="callout-content">
<ul>
<li><i class="bi-search " style="" role="img" aria-hidden="true"></i> Findable (Persistent identifiers, Indexed in a searchable resource).</li>
<li><i class="bi-hdd-fill " style="" role="img" aria-hidden="true"></i> Accessible (Software requirements, Open file formats).</li>
<li><i class="bi-diagram-2-fill " style="" role="img" aria-hidden="true"></i> Interoperable (rich metadata, standardized parameters).</li>
<li><i class="bi-recycle " style="" role="img" aria-hidden="true"></i> Reusable (descriptive metadata, Clear license and usage rights).</li>
</ul>
</div>
</div>
</div>
</div>
</section>
<section id="where-do-i-share-my-images" class="slide level2 center">
<h2>Where do I share my images?</h2>
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Repositories.png" style="width:100.0%" alt="A collection of logos representing various research data repositories, including OSF, Dryad, FRDR, DFDR, BioStudies, Zenodo, and Borealis. These repositories support open science by providing platforms for data sharing and preservation."></p>
<figcaption>Examples of generalist repositories</figcaption>
</figure>
</div>
</div>
</section>
<section id="features-of-some-generalist-repositories" class="slide level2 center">
<h2>Features of some generalist repositories</h2>
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Repositories_Features.webp" style="width:100.0%" alt="Comparison table of various research data repositories including Zenodo, Figshare, Dryad, Bioimage Archive, Image Data Repository, Broad Bioimage Benchmark Collection, and The Cell Painting Gallery. The table details repository URLs, qualifications, support for non-image data, size limits, costs, and metadata requirements.  "></p>
<figcaption>From <a href="https://www.nature.com/articles/s41592-023-01987-9/figures/7">Schmied. et al (2023)</a></figcaption>
</figure>
</div>
</div>
</section>
<section id="specialized-image-repositories" class="slide level2 center">
<h2>Specialized image repositories</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Idr.png" style="width:100.0%" alt="Screenshot of the Image Data Resource (IDR) webpage, a public repository of bio-image datasets from research. The interface shows options to search for datasets, filter by type, and access specific studies. The image highlights statistics such as 132 studies, over 14 million images, and 411 TB of data, with a grid preview of available microscopy images."></p>
<figcaption><a href="https://idr.openmicroscopy.org/">Image Data Resource (IDR)</a> and its <a href="https://github.com/IDR/idr-metadata">metadata scheme</a></figcaption>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Bioimage.png" style="width:100.0%" alt="Screenshot of the BioImage Archive website, a free online repository for biological images. The page describes the archive's role in storing and distributing imaging data from various modalities, provided they are linked to a peer-reviewed publication or have significant value. It highlights submission policies, integration with other bioimaging databases like EMPIAR and IDR, and adherence to FAIR and REMBI guidelines. The right panel features additional resources, including a quick tutorial and a promotional banner."></p>
<figcaption><a href="https://www.ebi.ac.uk/bioimage-archive/">BioImage Archive</a></figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="other-solutions" class="slide level2">
<h2>Other solutions</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Omero.png" style="width:100.0%" alt="Screenshot of the OMERO website homepage. The page features a dark-themed background with fluorescence microscopy images of cells. The OMERO logo is centered, followed by a description stating that OMERO is a secure central repository for managing, analyzing, and sharing microscopy images. It highlights compatibility with over 150 image file formats and supports access via desktop applications, web, or third-party software. A red button labeled 'Download Version 5.6' is prominently displayed."></p>
<figcaption><a href="https://www.openmicroscopy.org/omero/">OMERO</a></figcaption>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Irods.png" style="width:100.0%" alt="Screenshot of the iRODS website homepage. The page has a green background and presents iRODS as an open-source data management software. A circular diagram in the center highlights key iRODS capabilities, including Automated Ingest, Storage Tiering, Compliance, Publishing, Provenance, Integrity, Auditing, and Indexing. A 'Download iRODS' button is visible at the bottom."></p>
<figcaption><a href="https://irods.org/">IRODS</a></figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
<div style="text-align: left;font-size: 80%">
<p>These instances are installed in dedicated network (core facility) space for long-term storage and sharing.</p>
</div>
</section>
<section id="characteristics-of-shared-images" class="slide level2 center">
<h2>Characteristics of shared images</h2>
<div style="text-align: left;font-size: 60%">
<ul>
<li><span style="color:orange;">Raw</span> and <span style="color:green;">processed</span> images (processing operations should not alter the original image).</li>
<li><span style="color:green;">Uncompressed</span> and <span style="color:green;">lossless</span> open file formats.</li>
<li>OME-TIFF retain original (i.e .czi) metadata.</li>
<li>PNG (images with annotations) is preferred.</li>
<li>Accurate and descriptive (machine readable) <span style="color:green;">naming conventions</span> (i.e Subject_Group_Area_Marker): Use grouping factors to name images.</li>
<li>Creation of <span style="color:green;">README files</span> to contextualize and describe the content and methods used in the dataset.</li>
<li>Use CC-BY, or CC0 licenses.</li>
</ul>
</div>
<div style="text-align: left;font-size: 80%">
<div class="callout callout-tip callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Tip</strong></p>
</div>
<div class="callout-content">
<p>“We strongly discourage author statements that images ‘are available upon request’, as this has been shown to be inefficient” (<a href="https://www.nature.com/articles/s41592-023-01987-9">Schmied et al.&nbsp;2023</a>)</p>
</div>
</div>
</div>
</div>
</section></section>
<section>
<section id="sharing-reproducible-workflows" class="title-slide slide level1 agenda-slide center">
<h1>Sharing reproducible workflows</h1>
<div class="agenda-heading">
<p>Agenda</p>
</div>
<div class="agenda">
<ol>
<li><div class="agenda-inactive agenda-pre-active">
<p>Understanding BioImaging</p>
</div></li>
<li><div class="agenda-inactive agenda-pre-active">
<p>Planning and adquiring images</p>
</div></li>
<li><div class="agenda-inactive agenda-pre-active">
<p>Gathering image metadata</p>
</div></li>
<li><div class="agenda-inactive agenda-pre-active">
<p>Organizing and sharing image data</p>
</div></li>
<li><div class="agenda-active">
<p>Sharing reproducible workflows</p>
</div></li>
</ol>
</div>
</section>
<section id="image-processing-and-analysis" class="slide level2 center">
<h2>Image processing and analysis</h2>
<p>In any research workflow, the analysis of images must be:</p>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="3">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: flex-start;">
<h3 id="objective">Objective</h3>
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Objective.png" class="quarto-figure quarto-figure-center" style="width:70.0%" alt="Illustration of a book with a magnifying glass placed over it. The book has a yellow cover with a label on the top, and the magnifying glass has a red rim with a blue lens, symbolizing research, analysis, or study."></p>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: flex-start;">
<h3 id="reliable">Reliable</h3>
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Realible.webp" style="width:100.0%" alt="Close-up of a metallic carabiner with the word 'RELIABILITY' engraved on it. The carabiner is securely fastened with a golden locking mechanism and is attached to a strong fabric strap."></p>
<figcaption>From https://nexxis.com.au/</figcaption>
</figure>
</div>
</div>
</div>
<div id="third-column" class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: flex-start;">
<h3 id="reproducible">Reproducible</h3>
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Replication.jpg" style="width:100.0%" alt="Illustration of a researcher in a white lab coat painting on a canvas, which contains an image of himself painting on a canvas in an infinite recursive loop. The background is red, creating a striking contrast. The artwork symbolizes scientific replication, self-referential processes, and the iterative nature of research. "></p>
<figcaption>From https://med.stanford.edu/</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="image-analysis-from-rdm-perspective" class="slide level2 center">
<h2>Image analysis from RDM perspective</h2>
<div style="text-align: left;font-size: 80%">
<p>From an RDM perspective, analysis of biomedical images ideally entails:</p>
<ul>
<li><i class="bi-unlock " style="" role="img" aria-hidden="true"></i> Access to large datasets.</li>
<li><i class="bi-file-earmark-text-fill " style="" role="img" aria-hidden="true"></i> Records of image processing (code, scripts).</li>
<li><i class="bi-share-fill " style="" role="img" aria-hidden="true"></i> Sharing of results (images, tables, graphs).</li>
</ul>
</div>
<div style="text-align: left;font-size: 80%">
<div class="callout callout-tip callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Tip</strong></p>
</div>
<div class="callout-content">
<p>Accurate, descriptive <span style="color:green;">naming conventions and README</span> files with metadata, or codebooks are vital to assure the integrity of analysis pipelines.</p>
</div>
</div>
</div>
</div>
</section>
<section id="modular-approach-for-image-analysis" class="slide level2 center">
<h2>Modular approach for image analysis</h2>
<p>A <span style="color:green;">modular pipeline</span> establishes the main image analysis tasks into independent sub tasks.</p>
<div class="cell" data-reveal="true" data-fig-width="20" data-fig-height="9" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class=""><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart LR
A[Nuclei detection] --&gt; B[Quantification] 
B --&gt; C[Spatial analysis]
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
<div style="text-align: left;font-size: 80%">
<div class="callout callout-tip callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Tip</strong></p>
</div>
<div class="callout-content">
<p>Modularity allows to <span style="color:green;">construct complex analysis pipelines</span> from independent components that can function together. This promotes the <span style="text-decoration: underline;">reuse</span> of independent modules.</p>
</div>
</div>
</div>
</div>
</section>
<section id="hierarchy-of-image-analysis-tasks" class="slide level2 center">
<h2>Hierarchy of image analysis tasks</h2>
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Hierarchy.jpg" style="width:50.0%" alt="Diagram illustrating the hierarchy of image processing in biological research. The first section (A) differentiates between low-level signal-driven processes, such as filtering, segmentation, tracking, and registration, which generate outputs that are not directly interpretable, and high-level biology-driven processes like statistical analysis, inference, visualization, and categorization, which produce interpretable outputs. The second section (B) presents a workflow starting from image registration, followed by segmentation, tracking, filtering, and analysis, leading to visualization and interpretation."></p>
<figcaption>From <a href="https://doi.org/10.1242/jcs.254292">Driscoll, M &amp; Zaritsky, A. (2021)</a></figcaption>
</figure>
</div>
</div>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: left;font-size: 50%">
<p><strong>Low-level (technical knowledge)</strong></p>
<p>Transform images in other images or data:</p>
<ul>
<li>Preprocessing (e.g.&nbsp;deconvolution, noise removal)</li>
<li>Object detection and segmentation (e.g.&nbsp;cells, intracellular organelles)</li>
<li>Particle/object tracking</li>
</ul>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: left;font-size: 50%">
<p><strong>High-level (disciplinary knowledge)</strong></p>
<p>Transform outputs from low-level tasks into information with biological meaning:</p>
<ul>
<li>Data visualization</li>
<li>Fitting of statistical models</li>
<li>Statistical inference and uncertainty measurements</li>
</ul>
</div>
</div>
</div>
</div>
</section>
<section id="image-analysis-workflow" class="slide level2 center">
<h2>Image analysis workflow</h2>
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Graph_Workflows.webp" style="width:80.0%" alt="Diagram illustrating different image analysis workflows in biological research. The left panel shows microscopy images as input. The central panel presents three types of workflows: (a) an established workflow using predefined methods such as SynAct for synaptic activity analysis, (b) a new workflow composed of multiple processing steps like scripts, macros, pipelines, or plugins, and (c) a machine learning workflow utilizing neural networks for image classification. The right panel displays different outputs, including fluorescence intensity plots over time, segmented images, and classification counts."></p>
<figcaption>From <a href="https://www.nature.com/articles/s41592-023-01987-9">Schmied et al., 2023</a></figcaption>
</figure>
</div>
</div>
</section>
<section id="prioritize-openfree-software" class="slide level2">
<h2>Prioritize open/free software</h2>
<p>There are dozens of open/free options to analyze research images and share reproducible workflows:</p>
<div style="text-align: left;font-size: 80%">
<ul>
<li><a href="https://imagej.net/software/fiji/downloads">Fiji</a></li>
<li><a href="https://www.cellpose.org/">CellPose</a></li>
<li><a href="https://www.ilastik.org/">Ilastik</a></li>
<li><a href="https://www.openmicroscopy.org/omero/">OMERO</a></li>
<li><a href="https://qupath.github.io/">QuPath</a></li>
<li><a href="https://www.python.org/">Python</a></li>
<li><a href="https://napari.org/stable/">Napari</a></li>
<li><a href="https://imaging.usegalaxy.eu/">Galaxy</a> or https://usegalaxy.eu/</li>
</ul>
</div>
</section>
<section id="bioimage-as-resource-for-images-analysis-pipelines" class="slide level2 center">
<h2>BioImage as resource for images analysis pipelines</h2>
<div style="text-align: left;font-size: 80%">
<p><a href="https://doi.org/10.1101/2022.06.07.495102">BioImage.IO</a> is a community-driven AI model repository that provides access to pretrained AI models with a plethora of open/free software partners.</p>
</div>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/BioImage.io.png" style="width:100.0%" alt="Screenshot of the BioImage.IO website, showcasing the BioImage Model Zoo. The site offers access to advanced AI models for bioimage analysis with one-click integration. The page highlights compatibility with Fiji, ilastik, and ImJoy, and features a button labeled 'Explore the Zoo.' At the bottom, logos of community partners are displayed. The background contains a blue gradient with a stylized giraffe, trees, and animals."></p>
<figcaption><a href="https://bioimage.io/">BioImage</a></figcaption>
</figure>
</div>
</div>
</div>
<div id="Second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Bioimage_Examples.png" style="width:100.0%" alt="Screenshot of the BioImage Model Zoo, displaying multiple AI models for bioimage segmentation and analysis. Each model has a thumbnail, title, description, tags, and download count. Some models focus on nucleus, cell, and mitochondria segmentation using different deep learning techniques. The interface features icons for compatibility, licensing, and additional functionalities."></p>
<figcaption>From Bioimage</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="image-analysis-workflows" class="slide level2 center">
<h2>Image analysis workflows</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="3">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/CheckList_WorkflowsA.png" class="quarto-figure quarto-figure-center" style="width:100.0%"></p>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/CheckList_WorkflowsB.png" style="width:100.0%" alt="Checklist for the publication of image-analysis workflows, divided into three categories: established workflows, new workflows, and machine learning workflows. Each category lists criteria such as citation of workflow or platform, inclusion of example data, manual region of interest (ROI), exact version, public availability, rationale, and limitations. The checklist includes recommendations for minimal, recommended, and ideal documentation standards.  "></p>
<figcaption>From <a href="https://www.nature.com/articles/s41592-023-01987-9">Schmied et al., 2023</a></figcaption>
</figure>
</div>
</div>
</div>
<div id="third-column" class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/CheckList_WorkflowsC.png" class="quarto-figure quarto-figure-center" style="width:100.0%"></p>
</figure>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="why-to-use-code" class="slide level2 center">
<h2>Why to use code?</h2>
<div style="text-align: left;font-size: 80%">
<p>“The mouse is antisocial. The GUI is antisocial. So, what’s that mean? You have a problem to solve, and you solve it with a GUI. What do you have? A problem solved. But when you solve it with a command line interface in a scripting environment, you have an <span style="color:green;">artifact</span>. And all of the sudden, that artifact can be shared with someone” <strong>Jeffrey Snover</strong></p>
</div>
<div style="text-align: center;font-size: 150%">
<p>Use <span style="color:green;">CODE</span> not the <span style="color:red;">mouse</span>!</p>
</div>
</section>
<section id="overall" class="slide level2 center">
<h2>Overall</h2>
<div class="callout callout-tip callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Tip</strong></p>
</div>
<div class="callout-content">
<ul>
<li>Keep track of all changes and analysis procedures perform to images.</li>
<li>Organize and link files (using naming conventions) through processing to avoid errors.</li>
<li>Select final or intermediate results to share considering aspects such as storing space and long-term preservation.</li>
</ul>
</div>
</div>
</div>
</section>
<section id="publication-of-analysis-results" class="slide level2 center">
<h2>Publication of analysis results</h2>
<div style="text-align: left;font-size: 80%">
<p>Sharing of <span style="color:green;">research objects</span> in public, active research management repositories, like the <a href="https://osf.io/">Open Science Framework (OSF)</a> is an excellent strategy to promote open, reproducible research. Please consider sharing illustrative images, figures and tables used in publication.</p>
</div>
<div class="callout callout-tip callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Tip</strong></p>
</div>
<div class="callout-content">
<p>In general, these are lower resolution images/figures (.png) not used for analysis but for illustration in research reports (thesis, articles).</p>
</div>
</div>
</div>
</section>
<section id="examples-of-results-images" class="slide level2">
<h2>Examples of results images</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row quarto-layout-valign-center">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 60%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/CheckList_Formatting.webp" style="width:100.0%" alt="Guidelines for research image presentation, illustrating best practices for clarity and accuracy. Sections include: (a) focusing on relevant image content through cropping and resizing, (b) separating individual images, (c) indicating the origin of insets, (d) adjusting brightness and contrast appropriately, (e) ensuring channel annotations are visible against the background, (f) using grayscale channels to improve visibility of features, and (g) explaining channel values with calibration scales and indicators. "></p>
<figcaption>From <a href="https://www.nature.com/articles/s41592-023-01987-9">Schmied et al., 2023</a></figcaption>
</figure>
</div>
</div>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: left;font-size: 80%">
<p>Click to see examples of published results <a href="https://osf.io/mb257/">images</a> and <a href="https://osf.io/jxb4z/">figures</a>.</p>
</div>
</div>
</div>
</div>
</section>
<section id="data-should-be-shared" class="slide level2 center">
<h2>Data should be shared</h2>
<div style="text-align: left;font-size: 80%">
<p>‘Data is available on request’ statements in publications are found to be often unreliable in practice (<a href="https://onlinelibrary.wiley.com/doi/10.1111/jmi.13317">Schmidt et al., 2024</a>)</p>
</div>
<div class="callout callout-tip callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Open Science Principle</strong></p>
</div>
<div class="callout-content">
<p>Share the data as openly as possible and keep it only as closed as necessary.</p>
</div>
</div>
</div>
</section>
<section id="sharing-data-is-a-professional-responsibility" class="slide level2 center">
<h2><i class="bi-share-fill " style="" role="img" aria-hidden="true"></i> Sharing data is a professional responsibility</h2>
<p>Depositing a dataset in a repository is <span style="color:red;">NOT ONLY</span> an exercise in meeting the requirements of funding agencies and journals. It is an <span style="color:green;">ethical and professional responsibility</span> of researchers to ensure reproducible research, and the access and reuse of research data.</p>
</section>
<section id="therefore-research-needs-to-move-towards" class="slide level2 center">
<h2>Therefore, research needs to move towards</h2>
<ul>
<li><span style="color:green;">Competent</span> researchers in <span style="text-decoration: underline;">RDM</span> and data analysis.</li>
<li><span style="color:green;">Standardized approaches</span> to <span style="text-decoration: underline;">sharing</span> raw data and analysis code to support research findings.</li>
<li>Researchers with a <span style="color:green;">commitment to transparency</span> and best research practices to ensure <span style="text-decoration: underline;">integrity</span>.</li>
</ul>
</section>
<section id="section-2" class="slide level2 center">
<h2></h2>
<div style="text-align: center;font-size: 100%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/Repositories_Logo.png" alt="Logos of two Canadian research data repositories: FRDR (Federated Research Data Repository) and Borealis. The FRDR logo features a geometric pattern of yellow squares forming a diamond shape, with the repository name in black and gold text. The Borealis logo includes an artistic depiction of the Northern Lights over mountains and a lake, with the repository name in bold white text. " width="1000" height="220"></p>
<figcaption>Visit <a href="https://www.frdr-dfdr.ca/repo/">FRDR</a> or <a href="https://borealisdata.ca/">Borealis</a></figcaption>
</figure>
</div>
</div>
</section>
<section id="resources-and-support" class="slide level2 smaller">
<h2>Resources and support</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<section id="supporting-material" style="text-align: center;font-size: 80%">
<h3>Supporting material</h3>
<ul>
<li><a href="https://www.eurobioimaging.eu/data-services/fair-image-data/">FAIR image data</a></li>
<li><a href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/bioimage-metadata/tutorial.html">FAIR Bioimage Metadata</a></li>
<li><a href="https://www.frdr-dfdr.ca/docs/en/depositing_data/">FRDR documentation</a></li>
<li><a href="https://alliancecan.ca/en/services/research-data-management/learning-and-training/training-resources">Training resources</a> from the Alliance</li>
</ul>
</section>
</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div style="text-align: center;font-size: 80%">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/QR-code.png" alt="A QR code image that redirects to the presentation located in a GitHub repository." width="200" height="200"></p>
<figcaption>This presentation is available <a href="https://github.com/Alliance-RDM-GDR/RDM_BioImageFAIR">here</a> (English or French)</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
<h3 id="support-services">Support Services:</h3>
<div style="text-align: left;font-size: 90%">
<p>Contact us to ensure that your data is well prepared and can be effectively shared with the research community.</p>
<ul>
<li>Email: rdm-gdr@alliancecan.ca</li>
<li>https://www.frdr-dfdr.ca/repo/</li>
</ul>
</div>

</section></section>
    </div>
  <div class="quarto-auto-generated-content" style="display: none;">
<p><img src="images/alliance_logo.png" class="slide-logo"></p>
<div class="footer footer-default">
<p>Sharing images and reproducible workflows - FRDR curation team</p>
</div>
</div></div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="RDM_BioImageFAIR_en_files/libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/reveal-audio-recorder/recorder.js"></script>
  <script src="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/reveal-audio-recorder/RecordRTC.js"></script>
  <script src="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/reveal-audio-slideshow/plugin.js"></script>
  <script src="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/notes/notes.js"></script>
  <script src="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/search/search.js"></script>
  <script src="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="RDM_BioImageFAIR_en_files/libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': true,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'jumpToSlide': true,
'0': {"prefix":"getPath() + \"/playback/\""},
'1': {"suffix":".webm"},
'2': {"textToSpeechURL":null},
'3': {"defaultNotes":false},
'4': {"defaultText":false},
'5': {"advance":0},
'6': {"autoplay":false},
'7': {"defaultDuration":5},
'8': {"defaultAudios":true},
'9': {"defaultPlaybackRate":1},
'10': {"playerOpacity":0.05},
'11': {"playerStyle":"position: fixed; bottom: 4px; left: 25%; width: 50%; height:75px; z-index: 33;"},
'12': {"startAtFragment":false},
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleScrollView(event)\"><kbd>r</kbd> Scroll View Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"5\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: 'c/t',

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: false,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'fade',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1050,

        height: 700,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, RevealAudioRecorder, RevealAudioSlideshow, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const onCopySuccess = function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      }
      const getTextToCopy = function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
        text: getTextToCopy
      });
      clipboard.on('success', onCopySuccess);
      if (window.document.getElementById('quarto-embedded-source-code-modal')) {
        const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
          text: getTextToCopy,
          container: window.document.getElementById('quarto-embedded-source-code-modal')
        });
        clipboardModal.on('success', onCopySuccess);
      }
        var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
        var mailtoRegex = new RegExp(/^mailto:/);
          var filterRegex = new RegExp('/' + window.location.host + '/');
        var isInternal = (href) => {
            return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
        }
        // Inspect non-navigation links and adorn them if external
     	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
        for (var i=0; i<links.length; i++) {
          const link = links[i];
          if (!isInternal(link.href)) {
            // undo the damage that might have been done by quarto-nav.js in the case of
            // links that we want to consider external
            if (link.dataset.originalHref !== undefined) {
              link.href = link.dataset.originalHref;
            }
          }
        }
      function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
        const config = {
          allowHTML: true,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start',
        };
        if (contentFn) {
          config.content = contentFn;
        }
        if (onTriggerFn) {
          config.onTrigger = onTriggerFn;
        }
        if (onUntriggerFn) {
          config.onUntrigger = onUntriggerFn;
        }
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note) {
            return note.innerHTML;
          } else {
            return "";
          }
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>